lib/logger/lib/logger.ex

```diff
@@ -693,6 +693,53 @@ defmodule Logger do
     :gen_event.sync_notify(Logger, :flush)
   end
 
+  @doc """
+  Set logging level for given module.
+
+  This will take priority over the primary level set, so it can be
+  used to increase verbosity of some parts of the project.
```

Comment:
```suggestion
  used to increase or decrease verbosity of some parts of the project.
```

Reply:
I used only one, as I assumed that this was obvious that it can be used to decrease verbosity, what is less obvious is that it can also be used to increase it above the primary level.